english/check-phone/open-message/open-message.md

```diff
@@ -2,4 +2,10 @@ You open up the message, but nothing appears. Suddenly, the screen turns
 white. Two black dots appear, slowly morphing into eyes as the white of
 the screen takes over your whole field of vision. 
 
-The marshmallows are here.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.